concepts/observability-configurable-logging/07-multiple-logpipeline-investigation.md

```diff
@@ -0,0 +1,136 @@
+# Objective
+
+The telemetry operator comes with a pipeline configured to push logs to Loki. Additionally, Kyma users can define their own pipeline and push the logs to their own logging backend.
+
+This document investigates the side effects that may come up when, for example, the logging backend defined by  user stops working.
+
+## Setup
+
+The investigated configuration uses the `filesystem buffering`. When one of the log pipeline outputs fails, the logs are buffered on the file system. In contrast, the `in-memory` buffer has the disadvantage that if the Fluent Bit Pod is restarted, logs can be lost. During the test, we referred to the Fluent Bit knowledge article about [Buffering & Storage](https://docs.fluentbit.io/manual/administration/buffering-and-storage).
+
+The setup consisted of following items:
+1. Kyma with telemetry operator
+2. Two outputs, deployed in the Kyma cluster:
+    - Loki, which comes with Kyma
+    -[Mock server](./assets/logpipeline-invstigation/mock-server.yaml)
+3. [Log generator daemon set](./assets/logpipeline-invstigation/log-generator.yaml) to generate a huge amount of logs to fill the filesystem buffer faster
+4. A [Function](./assets/logpipeline-invstigation/func.js) to check if the logs are being delivered when one of the outputs is down
+5. To simulate failures, the port of the service was changed so that the DNS resolution keeps working but logs won't be delivered.
+
+## Test Cases
+
+### Case 1
+![a](./assets/logpipeline-invstigation/case-1/case-1.svg)
+
+Setup:
+1. One input with two outputs (limiting the maximum number of chunks in the `filesystem` buffer) without rewrite tags
+2. Two log pipelines:
+   - [Loki](./assets/logpipeline-invstigation/case-1/loki.yaml)
+   - [Mock server](./assets/logpipeline-invstigation/case-1/mockserver.yml)
+
+Result:
+- When one of the outputs is down, the filesystem buffer is filled up and only keeps 150M of latest data (old data is discarded first). Eventually, when the filesystem buffer at tail plugin is full, the output plugin for Loki stops as well.
```

Comment:
Be more precise what "output is down" means. Make clear that this is not related to the Fluent Bit configuration, but and outage of an external system.

Reply:
now I am using outage to make it more explicit.